tokio/src/sync/broadcast.rs

```diff
@@ -317,6 +351,9 @@ struct Shared<T> {
     /// Number of outstanding Sender handles.
     num_tx: AtomicUsize,
 
+    /// Number of outstanding weak Sender handles.
+    num_weak_tx: AtomicUsize,
```

Comment:
You are adding this only to add a `weak_count` method? You definitely do not need SeqCst for that.